### PR TITLE
Remove `state activate --replace`.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-816" title="DX-816" target="_blank">StoryDX-816</a>  Look into dropping `state activate --replace`
  </td></table>
  <br />
 

It was deprecated in favor of `state pull --set-project`.

I have tested `state install <package>` followed by `state pull --set-project <org/project>` where `project` is the name of the project given by the Platform in the link printed by the `state install` command.

### CODE REVIEW

File: cmd/state/internal/cmdtree/cmdtree.go [hunk 1]
```diff
@@ -161,7 +161,7 @@ func New(prime *primer.Values, args ...string) *CmdTree {
 		deployCmd,
 		scriptsCmd,
 		eventsCmd,
-		newPullCommand(prime),
+		newPullCommand(prime, globals),
```
Comment (mitchell-as) [inline cmd/state/internal/cmdtree/cmdtree.go:164]: This is already on master. This branch is based on a previously merged PR.

File: cmd/state/internal/cmdtree/pull.go [hunk 2]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/ActiveState/cli/internal/runners/pull"
 )
 
-func newPullCommand(prime *primer.Values) *captain.Command {
+func newPullCommand(prime *primer.Values, globals *globalOptions) *captain.Command {
```
Comment (mitchell-as) [inline cmd/state/internal/cmdtree/pull.go:10]: This is already on master. This branch is based on a previously merged PR.

File: cmd/state/internal/cmdtree/pull.go [hunk 3]
```diff
@@ -18,21 +18,16 @@ func newPullCommand(prime *primer.Values) *captain.Command {
...
-				Name:        "force",
-				Shorthand:   "",
-				Description: locale.Tl("flag_state_pull_force_description", "Force pulling specified project even if it is unrelated to checked out one"),
-				Value:       &params.Force,
-			},
 			{
 				Name:        "set-project",
 				Shorthand:   "",
-				Description: locale.Tl("flag_state_pull_set_project_description", "project even if it is unrelated to checked out one"),
+				Description: locale.Tl("flag_state_pull_set_project_description", "Pull from the specified project instead of the checked out one"),
```
Comment (mitchell-as) [inline cmd/state/internal/cmdtree/pull.go:24]: This is already on master. This branch is based on a previously merged PR.

File: cmd/state/internal/cmdtree/pull.go [hunk 4]
```diff
@@ -18,21 +18,16 @@ func newPullCommand(prime *primer.Values) *captain.Command {
...
 				Value:       &params.SetProject,
 			},
 		},
 		[]*captain.Argument{},
 		func(cmd *captain.Command, args []string) error {
+			params.Force = globals.NonInteractive
```
Comment (mitchell-as) [inline cmd/state/internal/cmdtree/pull.go:30]: This is already on master. This branch is based on a previously merged PR.

File: test/integration/pull_int_test.go [hunk 5]
```diff
@@ -43,6 +43,12 @@ func (suite *PullIntegrationTestSuite) TestPullSetProject() {
 
 	// update to related project
 	cp := ts.Spawn("pull", "--set-project", "ActiveState-CLI/small-python-fork")
+	cp.ExpectLongString("you may lose changes to your project")
```
Comment (mitchell-as) [inline test/integration/pull_int_test.go:46]: This is already on master. This branch is based on a previously merged PR.

File: test/integration/pull_int_test.go [hunk 6]
```diff
@@ -60,9 +66,9 @@ func (suite *PullIntegrationTestSuite) TestPullSetProjectUnrelated() {
 	cp.Expect("Pull aborted by user")
 	cp.ExpectNotExitCode(0)
 
-	cp = ts.Spawn("pull", "--force", "--set-project", "ActiveState-CLI/Python3")
-	cp.Expect("activestate.yaml has been updated")
-	cp.ExpectExitCode(0)
+	cp = ts.Spawn("pull", "--non-interactive", "--set-project", "ActiveState-CLI/Python3")
```
Comment (mitchell-as) [inline test/integration/pull_int_test.go:69]: This is already on master. This branch is based on a previously merged PR.

File: cmd/state/internal/cmdtree/pull.go [hunk 7]
```diff
@@ -18,21 +18,16 @@ func newPullCommand(prime *primer.Values) *captain.Command {
 		locale.Tl("pull_description", "Pull in the latest version of your project from the ActiveState Platform"),
 		prime,
 		[]*captain.Flag{
-			{
```
Comment (mitchell-as) [inline cmd/state/internal/cmdtree/pull.go:21]: This is already on master. This branch is based on a previously merged PR.

**Review by Naatan — APPROVED** [review]

**Pull request merged**